src/components/metadata/metadata.js

```diff
@@ -73,6 +75,8 @@ const MetaData = ({
   const onCloseClick = () => {
     // Deselecting a node automatically hides MetaData panel
     onToggleNodeSelected(null);
+    // and reset the URL to '/'
+    toFlowchartPage();
```

Comment:
This works great when the X icon is selected. though when you select the background of the Flowchart when the Metadata panel is open to close the panel the URL isn't reset. You'll need to reset it there as well.

Reply:
hey yeah i included this point now, thank you 😄